CHANGELOG.md

```diff
@@ -6,7 +6,27 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.2.0 (2021-04-13)
+
+This release adds major new features since the 2.1.1 release.
+We deem it moderate priority for upgrading.
+
+This release adds the skip-scan optimization, which significantly 
```

Comment:
should be `Skip Scan`

Reply:
I don't see that `Skip Scan` is used anywere. Either `SkipScan` or the [actual commit message](https://github.com/timescale/timescaledb/pull/3000) contains:
>This patch implements a skip-scan;

Reply:
SkipScan is the node name which follows camelcase convention for node names but for external messaging we decided to use "Skip Scan"